rebasebot/bot.py

```diff
@@ -427,13 +433,13 @@ def _init_working_dir(
     return gitwd
 
 
-def _manual_rebase_in_repo(repo: github3.github.repo.Repository) -> bool:
+def _manual_rebase_in_repo(repo: github3.github.repo.Repository) -> Tuple[bool, Optional[github3.pulls.PullRequest]]:
```

Comment:
Think there is no necessity to return a tuple there. Looking at this function I'm thinking that just a PR object might be returned.

Reply:
👍